geth/jail/jail.go

```diff
@@ -43,6 +48,85 @@ type Jail struct {
 	baseJSCode     string                     // JavaScript used to initialize all new cells with
 }
 
+// Copy returns a new JailCell instance with a new eventloop runtime associated with
+// the given cell.
+func (cell *JailCell) Copy() (common.JailCell, error) {
```

Comment:
Hm, the code here almost duplicates `NewJailCell`. Let's create `newJailCell(id, vm, lo)` and put the instantiation logic there leaving only `vm` and `lo` handling in `JailCell.Copy` and `Jail.NewJailCell`?